engine/src/main/java/org/hibernate/search/engine/search/query/impl/ProjectionHitAggregator.java

```diff
@@ -35,19 +34,19 @@
 	public ProjectionHitAggregator(
 			Function<DocumentReference, R> documentReferenceTransformer,
 			ObjectLoader<R, ?> objectLoader, Function<List<?>, T> hitTransformer,
-			int expectedHitSize, int expectedLoadCountPerHit) {
+			int expectedHitSize) {
 		this.documentReferenceTransformer = documentReferenceTransformer;
 		this.objectLoader = objectLoader;
 		this.hitTransformer = hitTransformer;
 		this.expectedHitSize = expectedHitSize;
-		this.expectedLoadCountPerHit = expectedLoadCountPerHit;
 	}
 
 	@Override
 	public void init(int expectedHitCount) {
 		referencesToLoad.clear();
 		hits.clear();
-		referencesToLoad.ensureCapacity( expectedHitCount * expectedLoadCountPerHit );
+		// FIXME it would be nice to bring this back, once we have clarified how to load the references
```

Comment:
If you want to merge this PR, could you please create a ticket for this? And maybe reference the ticket in this comment.